core/flyout_base.js

```diff
@@ -627,6 +628,7 @@ Blockly.Flyout.prototype.clearOldBlocks_ = function() {
     button.dispose();
   }
   this.buttons_.length = 0;
+  Blockly.Events.disable();
```

Comment:
This should be `enable()`.

Wrap everything between `disable` and here in a try/catch, and reenable events in a finally clause.  If something goes wrong you don't want to end up with events perma-disabled.

Reply:
Fixed by removing that piece.